src/index.js

```diff
@@ -3,14 +3,75 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./containers/App";
 import * as serviceWorker from "./serviceWorker";
-import "./i18n";
 import "bootstrap/dist/css/bootstrap.min.css";
+import { createStore, applyMiddleware, compose } from "redux";
+import rootReducer from "./reducers/rootReducer";
+import { Provider, useSelector } from "react-redux";
+import thunk from "redux-thunk";
+import {
+  reduxFirestore,
+  getFirestore,
+  createFirestoreInstance,
+} from "redux-firestore";
+import {
+  ReactReduxFirebaseProvider,
+  getFirebase,
+  isLoaded,
+} from "react-redux-firebase";
+import firebaseConfig from "./firebaseConfig";
+import firebase from "firebase/app";
+import { Container } from "react-bootstrap";
 
-ReactDOM.render(
-  <Suspense fallback={<div>Loading...</div>}>
-    <App />
-  </Suspense>,
+const store = createStore(
+  rootReducer,
+  compose(
+    applyMiddleware(thunk.withExtraArgument({ getFirebase, getFirestore })),
+    reduxFirestore(firebase, firebaseConfig)
+  )
+);
+
+// Construct required properties
```

Comment:
`Construct required properties for rrf` (this comment wasnt clear enough before)